Utils/ScriptValidator/Unit1.pas

```diff
@@ -44,12 +45,15 @@ procedure TForm1.FormCreate(Sender: TObject);
   OpenDialog1.InitialDir := ExtractFilePath(Application.ExeName);
 
   fScripting := TKMScripting.Create(nil);
+
+  DragAcceptFiles(Form1.Handle, True);

```

Comment:
Why do you address `Form1.Handle` instead of just `Handle` here?
Please change this and other places to just `Handle`